webapp/src/js/services/enketo.js

```diff
@@ -329,9 +346,19 @@ angular.module('inboxServices').service('Enketo',
       }
     };
 
+    var replaceMarkup = function(doc) {
+      $('.question-label, .question > .or-hint', doc.html).each(function () {
+        $(this).html(Markdown.basic($(this).html()));
```

Comment:
Didn't Markdown used to come with enketo? Why do we need to do it ourselves now?

Reply:
Enketo now does it in the form compilation stage so I've moved it into API.